rosplan_planning_system/src/PlanDispatch/EsterelPlanDispatcher.cpp

```diff
@@ -18,10 +18,6 @@ namespace KCL_rosplan {
         nh.getParam("timeout_actions", timeout_actions);
         nh.getParam("action_timeout_fraction", action_timeout_fraction);
 
-        std::string planTopic = "complete_plan";
-        nh.getParam("plan_topic", planTopic);
-        bl_end = (planTopic != "/robust_plan");
-
```

Comment:
Only one line was to be removed here - the one that sets the now removed "bl_end". I don't know if the other two lines are required, but they seem important! can you check?